spec/models/comment_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+describe Comment do 
+  let(:user){ build(:user) }
+  let(:comment){ build(:comment) }
+
+
+  # Happy
+  it "is valid with default attributes" do
+    expect(comment).to be_valid
+  end
+
+
+  it "responds to author assosiation" do
+    expect(comment).to respond_to(:author)
+  end
+
+  it "responds to likes assosiation" do
+    expect(comment).to respond_to(:likes)
+  end
+
+  # Sad
+  it "deleted if author is deleted" do
```

Comment:
Does this work? If not, it may work if you run user.save! at some point. I don't think there is technically a user to destroy unless you do